File: core/src/test/java/com/sys1yagi/core/testtool/TestAssetsUtils.java
package com.sys1yagi.core.testtool;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileInputStream;
import java.io.IOException;
import java.io.InputStreamReader;

public class TestAssetsUtils {

    private final static String ASSETS_PATH = "src/test/assets";

    public static String loadFromAssets(String name) {
        File file = new File(ASSETS_PATH, name);
        try (BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(file)))) {
            StringBuilder sb = new StringBuilder();
            String line;
            while ((line = br.readLine()) != null) {
                sb.append(line);
            }
            return sb.toString();
        } catch (IOException e) {
            return null;
        }
    }
}

File: core/src/test/java/com/sys1yagi/swipe/core/testtool/TestAssetsUtils.kt
package com.sys1yagi.swipe.core.testtool

import java.io.File
import java.io.FileInputStream

object TestAssetsUtils {

    private val ASSETS_PATH = "src/test/assets"

    fun loadFromAssets(name: String): String? {
        val file = File(ASSETS_PATH, name)
        return FileInputStream(file)
                .bufferedReader("utf-8")
                .readText()
    }
}
